src/EnhancedScatterChart.ts

```diff
@@ -957,8 +957,6 @@ export class EnhancedScatterChart implements IVisual {
         this.parseAxisSettings(settings.enableCategoryAxisCardSettings, colorHelper);
         this.parseAxisSettings(settings.enableValueAxisCardSettings, colorHelper);
 
-        settings.enableBackdropCardSettings.show.value = settings.enableBackdropCardSettings.show.value && !colorHelper.isHighContrast;
```

Comment:
what happens in contrast mode when you have a backdrop image now?

Reply:
Backdrop can not be switched on in high-contrast mode

Reply:
disable backdrop in high contrast mode